presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/MetastoreUtil.java

```diff
@@ -618,4 +627,85 @@ private static String getRenameErrorMessage(Path source, Path target)
     {
         return format("Error moving data files from %s to final location %s", source, target);
     }
+
+    public static List<String> convertPredicateToParts(TupleDomain<Column> effectivePredicate, List<Column> partitionColumns)
```

Comment:
Across the codebase we usually call "parts" the partition column values list. What this method creates - a filter or pattern. How about we call this method `createMetastorePartitionMathchingPattern`? or `createMetastorePartitionMathchingFilter`?

Reply:
As discussed, to keep it in line with the current code `parts` seems to be the better choice than `pattern`